src/components/auth/mentorLogin.tsx

```diff
@@ -22,10 +18,17 @@ class MentorLogin extends React.Component<RouteComponentProps<RouteProps>> {
       return;
     }
 
-    auth.mentorLogin({ Code: params.get('code'), State: params.get('state') }).then((mentorJwt: JwtAuthToken) => {
+    const linkedInAuthCode: ILinkedInAuthCode = {
+      Code: params.get('code'),
+      SendBirdAccessToken: 'sample-access-token',
+      State: params.get('state')
+    };
+
+    auth.mentorLogin(linkedInAuthCode).then((mentorJwt: JwtAuthToken) => {
       SessionService.StoreJwt(mentorJwt);
       const mentor: IMentor = JwtTokenHelper.decodeMentorJwt(mentorJwt.Token);
       SessionService.StoreMentor(mentor);
+      SendBirdClient.connect(mentor.Id!);
```

Comment:
Please correct me, but `Id` on `IMentor` is not optional or allows a `null` value so null check is probably not needed?
```suggestion
      SendBirdClient.connect(mentor.Id);
```

Reply:
you're right, it's not needed here. Thanks!